apps/application-system/api/src/app/modules/application/files/utils/aws.ts

```diff
@@ -0,0 +1,58 @@
+import * as AWS from 'aws-sdk'
+
+const s3 = new AWS.S3()
```

Comment:
Should this be a Nest module/service? To make it easier/cleaner to inject and stub.

Reply:
Fixed ✅  but want to pick your 🧠. Do you have any getlines that you follow when an utility is a service vs a module vs utility functions in the monorepo?

Reply:
My thinking is if it makes sense to dependency inject it, have it configured externally, or if I want a "simple" way to override/stub it in tests, then we should inject it using NestJS.

Usually applies to stuff that has external dependencies or side effects that we want to configure or stub in tests.

While pure utility functions don't necessarily need to be part of the NestJS DI container.